src/tests/extensions.rs

```diff
@@ -95,6 +95,7 @@ fn multi_dot_extension() {
     #[rustfmt::skip]
     let pass = [
         ("should resolve according to order of provided extensions", "./foo", "foo.ts"),
+        ("should resolve .d.ts for .ts", "./bar.ts", "bar.d.ts"),
```

Comment:
@coderabbitai Please help to find out the root case of the failing case.

Reply:
Then how to fix the issue exactly in this repo?

Reply:
The change line must be wrong, please check again.

Reply:
I've found out why the test case is failing at [`de19677` (#76)](https://github.com/unrs/unrs-resolver/pull/76/commits/de1967727e40eec073731f79414d1ba0abff693f), you can close your PR safely now.

Reply:
I told you to **close**, not **recreate**!